### ISSUE TITLE
power_inlet_filtered.stl cad issue?

### ISSUE THREAD

**Issue by leesafur** [VoronDesign/Voron-Trident#112]
Not enough thickness for the bottom hole?
![Screenshot 2023-03-28 171130 (2)](https://user-images.githubusercontent.com/42713329/228400779-9babff96-795f-4df5-91c9-c9a1205ee4dd.jpg)

**Comment by newphreak** [comment]
This is correct, 0.4mm difference from the middle and keystone skirt pieces on those bottom 3 partial hexagons.
https://cdn.discordapp.com/attachments/698953981023486012/1099131556087463996/skirt.gif

**Comment by stvptrsn** [comment]
Corrected in the Trident R2 release.
